.github/scripts/update-version.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -e
+
+sed -i "s/version=.*/version=$1/" gradle.properties
```

Comment:
can we add a comment saying this line will replace `version=.*` with `version=$1` where $1 is the input parameter? 
nit: maybe print error when $1 is not provided? 

i am thinking we can run update-version.sh independently not just used by release.yml.

Reply:
i ran it without $1, it will result to "version=".

Reply:
in that case subsequent "s/version=.*" search will fail